src/main/java/org/traccar/protocol/RuptelaProtocolDecoder.java

```diff
@@ -135,12 +150,22 @@ private void decodeParameter(Position position, int id, ByteBuf buf, int length)
                     position.set(Position.KEY_ALARM, Position.ALARM_ACCELERATION);
                 }
                 break;
+            case 150:
+                position.set(Position.KEY_OPERATOR, readValue(buf, length, false));
```

Comment:
What is the value here?

Reply:
Unique Mobile operator code - MCC and MNC on my devices

Reply:
I guess it's ok for now, but ideally we only store the name here. For MCC and MNC we have Network object.